winui/Masked-TextBox/Getting-Started.md

```diff
@@ -7,4 +7,85 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Getting Started with WinUI SfMaskedTextBox
\ No newline at end of file
+# Getting Started with WinUI SfMaskedTextBox
+
+This section explains the steps required to add the WinUI MaskedTextBox control and covers the basic features needed to get started with Syncfusion `MaskedTextBox` control.
+
```

Comment:
@Vignesh-SF3580 This section explains the steps required to add the WinUI MaskedTextBox control to get started with Syncfusion `MaskedTextBox` control.

Reply:
Changed the content like above.